bigflow/scaffold/scaffold_templates.py

```diff
@@ -0,0 +1,348 @@
+readme_template = '''# Run project
+
+### Requirements
+  1. [GCP SDK](https://cloud.google.com/sdk/docs).
+
+### Preparations
+  1. Log in using SDK (`gcloud auth application-default login`).
+  1. You need to create [Cloud Composer] (https://cloud.google.com/composer/docs/how-to/managing/creating#creating_a_new_environment).
+  1. You need to create GCS bucket `{project_name}` and inside this bucket create directories `beam_runner/staging` and `beam_runner/temp`.
+  1. In your deployment_config change key docker_repository to point your docker registry (we recommend to use one docker repository for all environments).
```

Comment:
Imo ten opis jest zbyt szczegółowy tu, pomyślcie jak trudno będzie potem to zmienić/poprawić w już wygenerowanych plikacj. Jeśli to jest instrukcja, to przenieśmy ją do docsów i podlinkujmy tu tylko